app/modules/events/controllers/eventControllers.js

```diff
@@ -149,6 +162,7 @@ angular.module('pcApp.events.controllers.event', [
                     description: angular.toJson(eventService.getEvent()[0]['description']).replace(/\"/g, ""),
                     startEventDate: angular.toJson(eventService.getEvent()[0]['date']).replace(/\"/g, ""),
                     endEventDate: angular.toJson(eventService.getEvent()[0]['date']).replace(/\"/g, ""),
+                    spatials: $scope.spatials.output,
```

Comment:
A get an error "$scope.spatials is undefined" when browsing the event creation page. Maybe because of this line.